post/views.py

```diff
@@ -62,3 +64,35 @@ def update_post(request, id):
             return redirect("/")
     context = {"form": form, "post": post, "success_message": success_message}
     return render(request, "post/update_post.html", context)
+
+class AddCommentView(CreateView):
+      model = Comment
+      form_class = CommentForm
+      template_name = 'post/add_comment.html'
+      success_url = reverse_lazy('')
```

Comment:
Hey, correct use for reverse_lazy is this reverse_lazy('home').

Reply:
done